lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -16099,6 +16099,12 @@ msgstr "En arrière"
 msgid "The end date must be greater than the start date"
 msgstr "La date de fin doit être supérieure à la date de début"
 
+msgid "Graph options"
+msgstr "Options de graphe"
+
+msgid "Display metric values tooltip"
+msgstr "Afficher l’info-bulle des valeurs de métriques"
```

Comment:
is it for 1 or more metrics here?

Reply:
Several metrics

Reply:
because "metric" is singular in English, and plural in French